src/pages/Home/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { SearchBar } from '../../components';
+// import { getResult } from '../../actions';
+
+class Home extends Component {
+    render() {
+         return (
+         <>
+         <h1>Welcome</h1>
+         Enter username:
+                 <SearchBar />
+        </>
+         )
+     }    
+}
+export default Home;
```

Comment:
This needs a few more options to work together with redux, getResult and SearchBar